### ISSUE TITLE
[Analytics] HLR: Back-End Grafana Dashboard for Submission Success/Errors

### ISSUE THREAD

**Issue by luke-gcio** [department-of-veterans-affairs/va.gov-team#10712]
## Issue Description
Gather metrics in Grafana

A new dashboard with:

1. Form Submissions
2. Form Error rates
3. rate of calls to Lighthouse
  api.decision_review.total
  api.decision_review.fail

## Acceptance Criteria
- [X] Grafana dashboard created ( http://grafana.vfs.va.gov/d/qCRjZjDMk/hlr-higher-level-review?orgId=1 )

**Comment by luke-gcio** [comment]
Final HLR fixes to integrate with LH took more of the sprint than expected, moving to sprint 29.

**Comment by edmangimelli** [comment]
Discussed / worked with Anna on this at length yesterday.

The way the code is currently, Grafana already has access to success and failure rates. Anna walked me through Grafana's web UI, and during that pairing session we created two graphs that are a good starting point for monitoring HLR success/failures/errors.

http://grafana.vfs.va.gov/d/qCRjZjDMk/hlr-higher-level-review?orgId=1

What monitoring will look like for HLR MVP:

tl;dr:  Keep an eye on Grafana. When we notice a problem, correlate with Sentry and investigate.

Grafana will show us total failures, what their HTTP statuses were, and when this happened.
From there, we can use Sentry to search for a DecisionReview Service error with the same status that happened around the same time, and then we should have enough info to troubleshoot (backtrace, error message, etc.)

What still needs to happen:
Once the big PR is merged ( https://github.com/department-of-veterans-affairs/vets-api/pull/4823 ) *and live in prod with an apikey*, send a dummy request to ensure that successes/failures are being recorded in the new dashboard.
